dev-python/python-ldap/python-ldap-3.4.0-r1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( pypy3 python3_{8..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Various LDAP-related Python modules"
+HOMEPAGE="https://www.python-ldap.org/en/latest/
+	https://pypi.org/project/python-ldap/
+	https://github.com/python-ldap/python-ldap"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/python-ldap/python-ldap.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-solaris"
+fi
+
+LICENSE="MIT PSF-2"
+SLOT="0"
+IUSE="examples sasl ssl"
+
+# < dep on openldap for bug #835637, ldap_r is gone
+RDEPEND="
+	>=dev-python/pyasn1-0.3.7[${PYTHON_USEDEP}]
+	>=dev-python/pyasn1-modules-0.1.5[${PYTHON_USEDEP}]
+	>=net-nds/openldap-2.6:=[sasl?,ssl?]
+"
+# We do not link against cyrus-sasl but we use some
+# of its headers during the build.
+DEPEND="
+	>=net-nds/openldap-2.6:=[sasl?,ssl?]
+	sasl? ( >=dev-libs/cyrus-sasl-2.1 )
+"
+
+distutils_enable_tests pytest
+distutils_enable_sphinx Doc
+
+PATCHES=( "${FILESDIR}"/${P}-openldap-2.6.patch )
```

Comment:
Could you include a link to upstream commit or PR?

Reply:
Sorry, there is no upstream commit or PR for that. There are only open issues regarding this problem. The used patch is from Fedora Rawhide Package sources. I'm not sure which commit Simon Pichugin refers in the patch comment.

Reply:
If in doubt, ask him. I can live with the patch not being merged upstream but it should be at least submitted there.

Reply:
I'll get in touch with him. But i don't know him and i have no ETA for an answer. Please be patient.

Reply:
Simon provided me the necessary information and I've added these to the Patch in the ebuild.